.husky/.prettierignore

```diff
@@ -0,0 +1,5 @@
+# Add files here to ignore them from prettier formatting
```

Comment:
@PepiPetrov Looks like the .husky folder got polluted again with some other files

Please clean it up so there is no diff in the PR for this folder